frontend/src/scenes/actions/Action.tsx

```diff
@@ -41,6 +43,10 @@ export function Action({ id }: { id?: ActionType['id'] } = {}): JSX.Element {
         return <NotFound object="action" />
     }
 
+    if (getAppContext()?.resource_access_control?.[AccessControlResourceType.Action] !== AccessControlLevel.Editor) {
```

Comment:
Will the backend return the action, though? Doesnt this negate part of the effect of having something like RBAC? If the backend were to not return anything - as it probably should -, we're probably falling in the `NotFound` clause above, right?